packages/presentation/lib/screens/splash/splash_screen.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:presentation/bloc/base/bloc_screen.dart';
+import 'package:presentation/const.dart';
+import 'package:presentation/navigation/base_page.dart';
+import 'package:presentation/screens/splash/splash_bloc.dart';
+
+class SplashScreen extends StatefulWidget {
+  const SplashScreen({super.key});
+
+  static const _routeName = '/SplashScreen';
+
+  static BasePage page() => BasePage(
+        key: const ValueKey(_routeName),
+        name: _routeName,
+        builder: (contex) => const SplashScreen(),
+      );
+
+  @override
+  State<SplashScreen> createState() => _SplashScreenState();
+}
+
+class _SplashScreenState extends BlocScreenState<SplashScreen, SplashBloc> {
+  @override
+  Widget build(BuildContext context) {
+    _setUiOverlayStyle();
+
+    return Container(
+      decoration: const BoxDecoration(
+        gradient: LinearGradient(
+          begin: Alignment.topCenter,
+          end: Alignment.bottomCenter,
+          colors: [
+            AppColors.splashScreenStartColorGradient,
+            AppColors.splashScreenEndColorGradient,
+          ],
```

Comment:
move this array to appColors and get it like AppColors.splashScreenGradient

Reply:
Fixed.